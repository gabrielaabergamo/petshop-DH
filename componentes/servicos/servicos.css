/*servi√ßos*/
.servicos-body{
    display: flex;
    justify-content: flex-start;
    align-items: flex-start;
}
.servicos-retangulo{
    background-color: darkgray;
    width: 266px;
    height: 264px;
    margin-left: 50px;
}

.servicos-info{
    margin-left: 100px;
}

.servicos-info-content{
    display: flex;
    justify-content: flex-start;
    align-items: flex-start;
    margin-bottom: 40px;
}

.servicos-info-content-title{
    text-transform: uppercase;
    margin: 0;
}

.servicos-info-content-preco{
    font-size: 17px;
    font-weight: 400;
    margin-top: 5px;
    margin-left: 10px;
}

.servicos-info-description{
    color: #757575;
    font-weight: 400;
    margin-bottom: 50px;
}

.servicos-info-btn{
    background-color: #FFC107;
    border: none;
    width: 245px;
    height: 32px;
    color: white;
}