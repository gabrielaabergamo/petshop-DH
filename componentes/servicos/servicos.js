import { formatter } from '../../js/formatmoney';
import { getData } from '../../js/db-request';

export const createMarkupServicos = async () => {
  const data = await getData();

  return data.map((servico) => `
    <div class="servicos-body">
        <div class="servicos-retangulo"></div>
        <div class="servicos-info">
            <div class="servicos-info__content">
                <h2 class="servicos-info__content-title">${servico.title}</h2> <small class="servicos-info__content-preco">${formatter.format(servico.price)}</small>
            </div>
            <p class="servicos-info__description">Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                sequi nesciunt. </p>
                <button class="servicos-info__btn">agendar</button>
        </div>
    </div>
    <img class="servicos-img" src="img/linha.png" alt="">
    `).join('');
};
