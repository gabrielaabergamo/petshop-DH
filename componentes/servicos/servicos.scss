.servicos{
    &-body{
        display: flex;
        justify-content: flex-start;
        align-items: flex-start;  
    }
    &-retangulo{
        background-color: darkgray;
        width: 266px;
        height: 264px;
        margin-left: 50px; 
    }
    &-info{
        margin-left: 100px;  
    }
    &-info__content{
        display: flex;
        justify-content: flex-start;
        align-items: flex-start;
        margin-bottom: 40px;  
    }
    &-info__content-title{
        text-transform: uppercase;
        margin: 0;
    }
    &-info__content-preco{
        font-size: 17px;
        font-weight: 400;
        margin-top: 5px;
        margin-left: 10px;   
    }
    &-info__description{
        color: #757575;
        font-weight: 400;
        margin-bottom: 50px;       
    }
    &-info__btn{
        background-color: #FFC107;
        border: none;
        width: 245px;
        height: 32px;
        color: white;       
    }
} 