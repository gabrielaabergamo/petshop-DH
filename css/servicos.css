
.servicos{
    margin-top: 70px;
    margin-bottom: 100px;
}
.servicos-title{
    text-align: center;
    color:#FFBE51;
    text-transform: uppercase;
    margin-bottom: 70px;
}
.servicos-container{
    display: flex;
    flex-wrap: wrap;
    align-items: center;
    margin-top: 30px;
}

.servicos-img{
    margin: 70px 0;
    width: 100%;
}

.servicos-img:last-of-type{ 
    display: none;
}
