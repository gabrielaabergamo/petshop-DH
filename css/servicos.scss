.servicos{
    margin-top: 70px;
    margin-bottom: 100px;
    &-title{
        text-align: center;
        color: $primary-color;
        text-transform: uppercase;
        margin-bottom: 70px; 
    }
    &-container{
        display: flex;
        flex-wrap: wrap;
        align-items: center;
        margin-top: 30px;
    }
    &-img{
        margin: 70px 0;
        width: 100%;
        &:last-of-type{
            display: none;
        }
    }
}
