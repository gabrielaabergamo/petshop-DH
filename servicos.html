<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="css/servicos.css">
    <title>Serviços</title>
</head>
<body>
    <header>
        <div class="container">
            <div class="logo">
                <img src="img/Petshop DH-logo-white 1.png" alt="Logo do Petshop">
            </div>

            <div class="links">
                <ul>
                    <li><a href="index.html">Home</a></li>
                    <li><a href="servicos.html">Serviços</a></li>
                    <li><a href="contato.html">Contato</a></li>
                    <li><a href="trabalhe-conosco.html">Trabalhe conosco</a></li>
                </ul>
            </div>
        </div>
    </header>

    <section class="servicos">
        <h1>Serviços</h1>
        <div class="container">
            <div>
                <div class="retangulo"></div>

                <div class="info">
                    <div>
                        <h2>banho completo</h2> <small>R$40,00</small>
                    </div>
                    <p>Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                        quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                        ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                        sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                        sequi nesciunt. </p>
                        <button>agendar</button>
                </div>
            </div>

            <img src="img/linha.png" alt="">

            <div>
                <div class="retangulo"></div>

                <div class="info">
                    <div>
                        <h2>banho completo</h2> <small>R$40,00</small>
                    </div>
                    <p>Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                        quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                        ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                        sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                        sequi nesciunt. </p>
                        <button>agendar</button>
                </div>
            </div>
            <img src="img/linha.png" alt="">

            <div>
                <div class="retangulo"></div>

                <div class="info">
                    <div>
                        <h2>banho completo</h2> <small>R$40,00</small>
                    </div>
                    <p>Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                        quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                        ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                        sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                        sequi nesciunt. </p>
                        <button>agendar</button>
                </div>
            </div>
        </div>
        
    </section>



    <footer>
        <div class="container">
            <div class="imagem-footer">
                <img src="img/logo-black 1.png" alt="">
            </div>
            <div class="texto-footer">
                <p>
                    CONTATO@PETSHOP.COM.BR <br>
                    (11) 98484-8484 (WHATSAPP) <br>
                    <br>
                    AVENIDA PAULISTA, 4242 <br>
                    BAIRRO CONSOLAÇÃO <br>
                    SÃO PAULO- SP <br>
                </p>
            </div>

            <div class="mapa-footer">
                <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d317715.7119190874!2d-0.3817828574308245!3d51.52873519860473!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47d8a00baf21de75%3A0x52963a5addd52a99!2sLondon%2C%20UK!5e0!3m2!1sen!2sbr!4v1631888347353!5m2!1sen!2sbr" width="466" height="257" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
            </div>
        </div>
        <p class="copyright">PETSHOP DH -  2021</p>
    </footer>
    
</body>
</html>