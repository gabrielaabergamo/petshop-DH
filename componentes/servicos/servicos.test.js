import {createMarkupServicos} from "./servicos"

test("createMarkupServicos should render cards", ()=>{
    expect(createMarkupServicos()).toMatchSnapshot()
})