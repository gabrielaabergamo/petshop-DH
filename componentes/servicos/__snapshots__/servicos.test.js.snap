// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createMarkupServicos should render cards 1`] = `
"
    <div class=\\"servicos-body\\">
        <div class=\\"servicos-retangulo\\"></div>

        <div class=\\"servicos-info\\">
            <div class=\\"servicos-info__content\\">
                <h2 class=\\"servicos-info__content-title\\">banho completo</h2> <small class=\\"servicos-info__content-preco\\">R$ 40,00</small>
            </div>
            <p class=\\"servicos-info__description\\">Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                sequi nesciunt. </p>
                <button class=\\"servicos-info__btn\\">agendar</button>
        </div>
    </div>
    <img class=\\"servicos-img\\" src=\\"img/linha.png\\" alt=\\"\\">
    
    <div class=\\"servicos-body\\">
        <div class=\\"servicos-retangulo\\"></div>

        <div class=\\"servicos-info\\">
            <div class=\\"servicos-info__content\\">
                <h2 class=\\"servicos-info__content-title\\">banho e tosa</h2> <small class=\\"servicos-info__content-preco\\">R$ 60,00</small>
            </div>
            <p class=\\"servicos-info__description\\">Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                sequi nesciunt. </p>
                <button class=\\"servicos-info__btn\\">agendar</button>
        </div>
    </div>
    <img class=\\"servicos-img\\" src=\\"img/linha.png\\" alt=\\"\\">
    
    <div class=\\"servicos-body\\">
        <div class=\\"servicos-retangulo\\"></div>

        <div class=\\"servicos-info\\">
            <div class=\\"servicos-info__content\\">
                <h2 class=\\"servicos-info__content-title\\">consulta veterinária</h2> <small class=\\"servicos-info__content-preco\\">R$ 150,00</small>
            </div>
            <p class=\\"servicos-info__description\\">Totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et <br>
                quasi architecto beatae vitae dicta sunt explicabo. Nemo enim <br>
                ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, <br>
                sed quia consequuntur magni dolores eos qui ratione voluptatem <br>
                sequi nesciunt. </p>
                <button class=\\"servicos-info__btn\\">agendar</button>
        </div>
    </div>
    <img class=\\"servicos-img\\" src=\\"img/linha.png\\" alt=\\"\\">
    "
`;
